integr8ly/integreatly-operator
# bump grafana version

# Issue link
https://issues.redhat.com/browse/MGDAPI-2671

# What
Bump grafana version to 3.10.4 

# Verification steps
Install RHOAM from master
Then run this branch to simulate an upgrade
Verify that Grafana upgrades to 3.10.4
Verify that dashboards still exist and look correct

Automated tests will cover the install scenario
